### ISSUE TITLE
How to get the label s of BVCC test wav?

### ISSUE THREAD

**Issue by jiusansan222** [nii-yamagishilab/mos-finetune-ssl#2]
How can I get BVCC test dataset? I only found data and label of train and val in VoiceMOS codalab, but I don't know how to get the label s of test wav.  Is it opensource?

**Comment by ecooper7** [comment]
Hi, the test set labels are available on the CodaLab page here:  https://codalab.lisn.upsaclay.fr/competitions/695#participate  (at the third bullet point).  This link is temporary; we are preparing a more permanent link to the data and will post it here on the README once it is ready.

**Comment by jiusansan222** [comment]
This website doesn't seem to work, is there any other method to get the labels? Could you E-mail me a copy? Thanks!

**Comment by ecooper7** [comment]
Hi, the new permanent link for the dataset is here:  https://zenodo.org/record/6572573#.YzPVui8RqLc
